util/secret/certificates.go

```diff
@@ -215,16 +233,25 @@ func (c Certificates) Generate() error {
 				continue
 			case ServiceAccount:
 				generator = generateServiceAccountKeys
+			case ClusterAPIEtcdClient:
+				if etcdCA == nil {
```

Comment:
What do you think about adding a comment saying the EtcdCA certificate must come before ClusterAPIEtcdClient certificate in the Certificates type?

Reply:
Replaced with something less brittle.